GVFS/GVFS/CommandLine/GVFSVerb.cs

```diff
@@ -85,8 +85,8 @@ public string InternalParameters
         public string ServicePipeName
         {
             get
-            {
-                return this.ServiceName + ".Pipe";
+            {
+                return GVFSPlatform.Instance.GetGVFSServiceNamedPipeName();
```

Comment:
I suspect the change from `this.ServiceName + ".Pipe"` to `GVFSPlatform.Instance.GetGVFSServiceNamedPipeName` will break some things on Windows because during the functionals tests the test service's pipe is named `Test.GVFS.Service`.

Perhaps `GetGVFSServiceNamedPipeName` should take a service name as a parameter?

Reply:
I missed the test service! thanks. I will update to take in the service name.